opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -72,27 +72,40 @@ def unwrap(obj, attr: str):
         setattr(obj, attr, func.__wrapped__)
 
 
-def get_token_context_span_kind(env, getter):
-    """Based on presence of active span, extracts context and initializes token and span_kind
+def start_internal_or_server_span(
```

Comment:
should be private function so prefix with `_`

Reply:
done